csrankings-g.csv

```diff
@@ -563,6 +563,7 @@ Gonçalo Nuno Gomes Tavares,Universidade de Lisboa,https://fenix.tecnico.ulisboa
 Gonçalo Tavares,Universidade de Lisboa,https://fenix.tecnico.ulisboa.pt/homepage/ist13269,mRRctaMAAAAJ
 Gong Cheng 0001,Nanjing University,http://ws.nju.edu.cn/~gcheng,_ncKAiwAAAAJ
 Gong Gu,University of Tennessee,http://web.eecs.utk.edu/~ggu1,A2-AVVcAAAAJ
+Gongming Zhao,USTC,http://staff.ustc.edu.cn/~zgm1993/,Oj8TNw4AAAAJ

```

Comment:
Research faculty (not TT) are not eligible for inclusion; please remove.

Reply:
We have removed the not eligible research faculty. Thanks.